### PR TITLE
MAISTRA-1504 Fix import k8s.io/kubernetes/pkg/apis/rbac

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -25,7 +25,6 @@ require (
 	k8s.io/code-generator v0.17.1
 	k8s.io/helm v2.16.1+incompatible
 	k8s.io/kubectl v0.0.0
-	k8s.io/kubernetes v1.16.2
```
Comment (jwendell) [inline go.mod:28]: It would be good if you run `go mod tidy` and commit the results of go.sum as well.

Comment (luksa) [reply go.mod:28]: I have run it, but it produces no changes in go.sum.

Comment (luksa) [reply go.mod:28]: I have run it, but it produces no changes in go.sum.

**Review by rcernich — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (jwendell) [reply go.mod:28]: ah ok, it must be other modules depending on it.

**Review by jwendell — APPROVED** [review]

**Pull request merged**